### ISSUE TITLE
deploying on heroku

### ISSUE THREAD

**Issue by apexcodexd** [itzg/docker-minecraft-bedrock-server#138]
is there any way we can deploy this on heroku??

**Comment by itzg** [comment]
I don't think so since Heroku is intended for deploying web based services. The bedrock server/client uses a proprietary UDP protocol.
